files/image_config/monit/memory_checker

```diff
@@ -0,0 +1,90 @@
+#!/usr/bin/env python
```

Comment:
Suggest changing to `#!/usr/bin/env python3` until we fully remove Python 2. This will ensure the script is Python 3-compliant and we will not encounter unexpected issues when we remove Python 2.

Reply:
Updated. Thanks Joe for reminding this issue again!